modules/telco-core-cpu-partitioning-and-performance-tuning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-cpu-partitioning-and-performance-tuning_{context}"]
+= CPU partitioning and performance tuning
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+CPU partitioning separates sensitive workloads from general-purpose tasks, interrupts, and driver work queues, improving performance and reducing latency.
+The CPUs allocated to those auxiliary processes are referred to as reserved in the following sections.
+In Hyper-Threaded systems, a CPU is one hyper-thread.
+
+Limits and requirements::
+* The operating system needs a certain amount of CPU to perform all the support tasks including kernel networking.
```

Comment:
So that it doesn't read like the tasks include kernel networking.
```suggestion
* The operating system needs a certain amount of CPU to perform all the support tasks, including kernel networking.
```